jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -211,11 +211,15 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
             the native data unit. If True, subsets are returned in the spectral
             axis display unit set in the Unit Conversion plugin.
 
-        return_sky_region : bool or None, optional
-            If None (default) or True, then the returned region will be ``SkyRegion`` if the
-            configuration is Imviz and the data is aligned by WCS, or if the configuration
-            is Cubeviz and the data has a WCS'. If set to False, a ``PixelRegion`` object will
-            be returned.
+        wrt_data : bool, str,  or None
```

Comment:
'If aligned by WCS and ``wrt_data`` is not False, return ``SkyRegion`` object.' wouldn't this case reutrn a pixel region?

Reply:
It would return a `PixelRegion` object if aligned by WCS and `wrt_data` was explicitly False (i.e. not None, True, or a string).